benchmarks/operator_benchmark/pt/qactivation_test.py

```diff
@@ -6,51 +6,58 @@
 import random
 
 import torch
-import torch.quantization as tq
 import torch.nn.quantized as nnq
 
 import operator_benchmark as op_bench
 
 r"""Microbenchmarks for the quantized activations."""
 
-qrelu_configs = op_bench.cross_product_configs(
+qactivation_long_configs = op_bench.cross_product_configs(
     dims=(
-        (1,), (1, 1), (1, 1, 1),     # Single element
-        (2, 1), (1, 2),              # Rank=2 row-/col-major
-        (3, 4, 5),                   # Rank=3
-        (1, 3, 4, 5), (2, 3, 4, 5),  # Rank=4, batch=1, batch>1
-        (4, 1, 1, 1),                # Rank=4, all other single dimensions
+        # VGG-16 relu's with original shape: (-1, 3, 224, 224)
+        ( 64, 224, 224),  # ReLU-1   # noqa
+        (128, 112, 112),  # ReLU-6   # noqa
+        (256,  56,  56),  # ReLU-11  # noqa
+        (512,  28,  28),  # ReLU-18  # noqa
+        (512,  14,  14),  # ReLU-25  # noqa
+        # Batch = 16
+        (16,  64, 224, 224),  # ReLU-1   # noqa
```

Comment:
For which operator are these configurations used?
If it is for relu/relu6 this is an overkill. I would have just 2 configs: One in 3D and one in 4D.

Reply:
One 3d and one 4d are already part of the short test below. These are longer tests for the ops -- the thing is that we noticed that the implementation is different depending on the size of the input, which is why I added all ReLU layers from the VGG. This is in total 40 tests, which translates to ~6.5 minute run (runs once every 8 hours).

I think we should keep different dimensions for benchmarking -- alternatively, we should always test with the same size.